portfolio/src/main/webapp/script.js

```diff
@@ -50,10 +50,21 @@ function createImg(imageName) {
 
 
 function getHelloMsg(){
-    fetch('/data').then(response => response.text()).then((massage) => {
-    document.getElementById('massage-container').innerText = massage;
+    fetch('/data').then(response => response.json()).then((massages) => {
+    const messageListElement = document.getElementById('massage-container');
+    messageListElement.innerText = '';
+    messageListElement.appendChild(createListElement(massages[2]));
```

Comment:
Now noticed that you actually want to use the messages you're inputting in initialization here.
This however isn't good practice: the reader doesn't know what indexes 2, 0 and 1 stand for, and imagine these change? Creating this type of dependency is risky. Also I believe you'd like to update your messages at some point, so this has to be generalized.

I'd suggest iterating on the messages (with a loop) and adding the messages one after the other. This means you'd have to keep them in order in your constructor as well, yet this is far less error prone.

Reply:
Done

Reply:
The changes you made aren't pushed (still looking at the previous version).